unix/vncpasswd/vncpasswd.cxx

```diff
@@ -109,10 +151,22 @@ static std::vector<uint8_t> readpassword() {
     std::string first = passwd;
     if (first.size() < 6) {
       if (first.empty()) {
-        fprintf(stderr,"Password not changed\n");
+        fprintf(stderr,_("Password not changed\n"));
         exit(1);
       }
-      fprintf(stderr,"Password must be at least 6 characters - try again\n");
+      fprintf(stderr,_("Password must be at least 6 characters - try again\n"));
```

Comment:
This is now policy that could conflict with what pwquality wants. It should be removed if we always use pwquality, or used as a fallback if pwquality ends up being optional.